### ISSUE TITLE
skypack error on v3.12.0

### ISSUE THREAD

**Issue by TillaTheHun0** [colinhacks/zod#965]
I tried importing the most recent version of skypack and received an error:
```
[Package Error] "zod@v3.12.0" could not be built. 
[1/5] Verifying package is valid…
[2/5] Installing dependencies from npm…
No matching version found for zod@3.12.0.
```

Reproduce:
Go to https://www.skypack.dev/view/zod and click "Open in Codepen" and view the console logs

version 3.11.6 works as expected with Skypack via `import * as zod from 'https://cdn.skypack.dev/zod@3.11.6'`, so not sure if something changed in 3.12.0

**Comment by JacobWeisenburger** [comment]
As of yesterday I hear that @colinhacks was on vacation, so I'm guessing he will take a look as soon as he is back. Not sure when that will be though.

**Comment by TillaTheHun0** [comment]
Thanks @JacobWeisenburger for the heads up. We've hardpinned to 3.11.6 for now.

**Comment by colinhacks** [comment]
Fixed in 3.12.1 even though I made no changes...I think Skypack might have 500d and cached the failure or something.
